lapack/gonum/dgesdc2.go

```diff
@@ -0,0 +1,50 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gonum
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/blas/blas64"
+)
+
+// Dgesdc2 solves a system of linear equations
+//   A * X = scale* RHS
+// with a general N-by-N matrix A using the LU factorization with
+// complete pivoting computed by Dgetc2.
+func (impl Implementation) Dgesdc2(n int, a []float64, lda int, rhs []float64, ipiv, jpiv []int, scale float64) {
+	const eps = dlamchP
+	var smlnum, _ = impl.Dlabad(dlamchS/eps, 1./(dlamchS/eps))
+	if len(a) < (n-1)*lda+n {
+		panic(shortA)
+	}
+	//Apply permutations IPIV to RHS
```

Comment:
`// Apply permutations ipiv to RHS.` (final dot and space between comment marks to distinguish from directives). Though I'm not sure if this comment is needed (@vladimir-ch?).

Also separate this comment from the conditional panic by a blank line.